ui/core/components/individual_sim_ui/apl_values.ts

```diff
@@ -1169,12 +1192,25 @@ const valueKindFactories: { [f in ValidAPLValueKind]: ValueKindConfig<APLValueIm
 		newValue: APLValueAuraInternalCooldown.create,
 		fields: [AplHelpers.unitFieldConfig('sourceUnit', 'aura_sources'), AplHelpers.actionIdFieldConfig('auraId', 'icd_auras', 'sourceUnit')],
 	}),
-	auraIcdIsReadyWithReactionTime: inputBuilder({
-		label: 'Aura ICD Is Ready (with Reaction Time)',
+	auraIcdIsReady: inputBuilder({
+		label: 'Aura ICD Is Ready',
 		submenu: ['Aura'],
 		shortDescription:
 			"<b>True</b> if the aura's ICD is currently ready OR it was put on CD recently, within the player's reaction time (configured in Settings), otherwise <b>False</b>.",
-		newValue: APLValueAuraICDIsReadyWithReactionTime.create,
+		newValue: () => APLValueAuraICDIsReady.create({ includeReactionTime: true }),
+		fields: [
+			AplHelpers.unitFieldConfig('sourceUnit', 'aura_sources'),
+			AplHelpers.actionIdFieldConfig('auraId', 'icd_auras', 'sourceUnit'),
+			AplHelpers.booleanFieldConfig('includeReactionTime', 'Include Reaction Time', {
+				labelTooltip: 'If checked, will use the configured reaction time.',
+			}),
```

Comment:
Since you are copy-pasting this same text each time, I would recommend making a helper for it in `apl_helpers.tsx`:
```
export function reactionTimeCheckbox(): APLPickerBuilderFieldConfig<any, any> {
    return booleanFieldConfig('includeReactionTime', 'Include Reaction Time', {
        labelTooltip: 'If checked, will use the configured reaction time.',
    });
}
```